package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "gulp-manifest",
-  "version": "0.0.7",
+  "version": "0.0.8",
```

Comment:
Some time passed by since this PR. We already passed that version number :wink:

@benediktvaldez would you please stash this change, so that your fix can be merged?